src/NUnitConsole/nunit-console/ConsoleRunner.cs

```diff
@@ -302,7 +302,18 @@ public static TestPackage MakeTestPackage(ConsoleOptions options)
             if (options.Verbose)
                 package.AddSetting("Verbose", true);
 
+            if (options.DebugTests)
+            {
+                package.AddSetting(PackageSettings.DebugTests, true);
+
+                if (options.NumWorkers < 0)
+                    package.AddSetting(PackageSettings.NumberOfTestWorkers, 0);
```

Comment:
Sure, go ahead. I'll look at creating a utility method for testing the settings dictionary in a future PR.